drivers/sx126x/sx126x_netdev.c

```diff
@@ -30,6 +30,10 @@
 #include "sx126x.h"
 #include "sx126x_netdev.h"
 
+#if IS_USED(MODULE_SX126X_PERIPH)
+#include "board.h"
```

Comment:
why do we need a board dependency from the driver? this seems weird

Reply:
This was part of the RF switched. Removed.

Reply:
Did you forget to push?